addons/procurement/procurement.py

```diff
@@ -44,7 +44,7 @@ class procurement_group(osv.osv):
         'name': fields.char('Reference', required=True),
         'move_type': fields.selection([
             ('direct', 'Partial'), ('one', 'All at once')],
-            'Delivery Method', required=True),
+            'Delivery Type', required=True),
```

Comment:
move_type does not necessarily have to do with outgoing pickings (delivery). Maybe we can call it differently.  (Reservation type / ...)